.github/workflows/pre-compile_llvm.yml

```diff
@@ -3,7 +3,7 @@ name: Pre-compile llvm
 on:
   workflow_dispatch:
   push:
-    branches: [ release_100, release_11x, release_12x ]
+    branches: [ release_100, release_11x, release_12x, release_13x ]
```

Comment:
Seems like release_100 could go?

Reply:
Some time ago we were thinking to remove it, but community wanted to leave it in case someone wants to test a commit against release_100 build. However no actual test is being triggered on release_100 branch, we remove that

Reply:
OK - happy to leave it in.